### PR TITLE
Updated velero chart to 2.15.0

### LINKED ISSUE #289: [1] Update velero helm chart to support S3 region
**Is your feature request related to a problem? Please describe.**
Update velero/restic to later version. To be able to set up this repo on [OVH](www.ovh.com) we need to update velero to a version later than `1.3.0` (We are running `1.2.0` and latest as of writing is `1.5.0`)
This is because restic doesn't support the `region` config on our current version [see this](https://github.com/vmware-tanzu/velero/issues/2237)

**Describe the solution you'd like**
Update the velero helm chart to a version that at least uses velero/restic version `1.3.0`

**Describe alternatives you've considered**
Not supporting cluster backup on OVH

**Additional context**
N/A

**Definition of done:**
When you can deploy velero and it can successfully take a backup of the cluster.

### PR DESCRIPTION
**Pull request by Xartos**
**What this PR does / why we need it**:

To support S3 region we need to upgrade velero to use at least version 1.3.0. 

**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: 
fixes #289 

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [x] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

**Review by OlleLarsson — APPROVED** [review]
LGTM. I assume upgrading crds doesn't pose any issues, right?

**Comment by Xartos** [comment]
> LGTM. I assume upgrading crds doesn't pose any issues, right?

Yes, I have tested to do an upgrade from our current version without any issues.

File: WIP-CHANGELOG.md [hunk 1]
```diff
@@ -13,3 +13,4 @@
 ### Changed
 
 - ClusterIssuers are used instead of Issuers
+- Updated velero chart to 2.15.0
```
Comment (OlleLarsson) [inline WIP-CHANGELOG.md:16]: + CRDs?

Comment (viktor-f) [reply WIP-CHANGELOG.md:16]: You could also note which version of velero we have upgraded to

Comment (Xartos) [reply WIP-CHANGELOG.md:16]: Yea, I thought the CRDs as a part of the chart. But you are right, for us they are a separate thing.

**Review by viktor-f — APPROVED** [review]

**Pull request merged**